### ISSUE TITLE
Tabbed browser example

### ISSUE THREAD

**Issue by aolko** [hgourvest/dcef3#28]
Any example for tabbed browser? (Overriding OnBeforePopup, creating a new tab, closing a tab)

**Comment by felipedaragon** [comment]
I know you are probably after a simple example, but if you would like to see a more complex one, check my project https://github.com/felipedaragon/sandcat

**Comment by aolko** [comment]
that's...lua
p.s. too complex, didn't get it _at all_
btw, how do you hook up sciter?

**Comment by felipedaragon** [comment]
No, only 6% is Lua, 78% is Pascal and using DCEF :) For Sciter, it's easy: https://github.com/da-baranov/SciDe

**Comment by awenewa** [comment]
https://github.com/bccsafe/DcefBrowser
